holoviews/element/geom.py

```diff
@@ -52,6 +52,35 @@ class VectorField(Selection2DExpr, Geometry):
     vdims = param.List(default=[Dimension('Angle', cyclic=True, range=(0,2*np.pi)),
                                 Dimension('Magnitude')], bounds=(1, None))
 
+    @classmethod
+    def from_uv(cls, data, kdims=None, vdims=None, **params):
+        if isinstance(data, tuple):
+            xs, ys, us, vs = data
+        else:
+            us = data[vdims[0]]
+            vs = data[vdims[1]]
+
+        uv_magnitudes = np.hypot(us, vs)  # unscaled
+        radians = np.pi / 2 - np.arctan2(us, vs)
```

Comment:
Why not `radians = np.pi / 2 - np.arctan2(-us, -vs)`?

Reply:
Vectorfield conventions are "to"

Reply:
Should we add a comment about this?

Reply:
```python
import numpy as np

x = 0
y = 5
print(
    np.rad2deg(np.arctan2(-y, -x)),  # meteorological (from)
    np.rad2deg(np.arctan2(-y, -x) + np.pi),  # oceanographic (to)
    np.rad2deg(np.arctan2(x, y)), # mathematical
)
```
-90.0 90.0 0.0